python/runtime/common/py_NoiseModel.cpp

```diff
@@ -43,6 +43,31 @@ void extractKrausData(py::buffer_info &info, complex *data) {
          sizeof(complex) * (info.shape[0] * info.shape[1]));
 }
 
+void extractRealData(py::buffer_info &info, real* data)  {
+  if (info.format != py::format_descriptor<real>::format())
+    throw std::runtime_error(
+        "Incompatible buffer format, must be np.complex128.");
+
+  if (info.ndim != 2)
+    throw std::runtime_error("Incompatible buffer shape.");
+
+  constexpr bool rowMajor = true;
```

Comment:
Why make it constexpr?

Reply:
I did not make it constexpr, but rather just re-did copied the same function that they've used 
I need to figure out a better template-magic to not have 2 copies but use the same